src/Auth0.OidcClient.Core/Auth0ClientBase.cs

```diff
@@ -21,19 +21,26 @@ public abstract class Auth0ClientBase : IAuth0Client
         private readonly Auth0ClientOptions _options;
         private readonly string _userAgent;
         private IdentityModel.OidcClient.OidcClient _oidcClient;
+        private IdentityModel.OidcClient.OidcClient OidcClient
+        {
+            get
+            {
+                return _oidcClient ?? (_oidcClient = new IdentityModel.OidcClient.OidcClient(CreateOidcClientOptions(_options)));
+            }
+        }
 
         /// <summary>
-        /// Creates a new instance of the Auth0 OIDC Client.
+        /// Create a new instance of <see cref="Auth0ClientBase"/>.
         /// </summary>
-        /// <param name="options">The <see cref="Auth0ClientOptions"/> specifying the configuration for the Auth0 OIDC Client.</param>
-        /// <param name="platformName">The platform name that forms part of the user-agent when communicating with Auth0 servers.</param>
-        public Auth0ClientBase(Auth0ClientOptions options, string platformName)
+        /// <param name="options"><see cref="Auth0ClientOptions"/> specifying the configuration options for this client.</param>
+        /// <param name="platformName">Platform name that forms part of the user-agent when communicating with Auth0 servers.</param>
+        protected Auth0ClientBase(Auth0ClientOptions options, string platformName)
```

Comment:
Linter warning addressed.  Abstract types should not have public constructors, protected is recommended.